src/components/editUserComponents/UploadImage.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react"
+import { Container, Row, Col, Form, Button } from "react-bootstrap"
+import axios from "axios";
+import authService from "../../services/auth.service";
+
+function UploadImage() {
+    const [id, setID] = useState()
+    const [vaccine, setVaccine] = useState()
+    const uuid_creds = authService.getCurrentUser().uuid
+    const handleIDUpload = (e) => {
+        const fd = new FormData()
+        fd.append("uuid", uuid_creds)
+        fd.append('id',id)
+        axios({
+            url: 'http://localhost:8080/api/image/id',
+            method: "POST",
+            headers: {
+                "content-type": "multipart/form-data"
+            },
+            data: fd
+        }).then((res) => {
+            console.log(res)
+        })
+    }
+    const handleIDChange = (e) => {
+        const id = e.target.files[0];
+        setID(id)
+    }
+
+    const handleVaccineUpload = (e) => {
+        const fd = new FormData()
+        fd.append("uuid", uuid_creds)
```

Comment:
let's standardize our `"` and `'`.
use `"` for html elements & component attributes then `'` for all the others